index.html

```diff
@@ -0,0 +1,234 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Online Course capstone</title>
+    <meta
+      name="description"
+      content="Online Course for capstone project of html, css and javascript"
+    />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="./bootstrap/css/bootstrap.css"
+    />
+    <link rel="stylesheet" type="text/css" href="./Css/style.css" />
+    <link
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css"
+      rel="stylesheet"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header class="header col-12">
+      <nav class="d-block p-2 bg-dark d-none d-lg-block">
+        <ul class="nav justify-content-end">
+          <li class="nav-link">
+            <a
+              class="nav-link active"
+              aria-current="page"
+              href="https://www.linkedin.com/in/karam-fouad-179830214/"
+              rel="noreferrer noopener"
+            >
+              <i class="fab fa-linkedin-in"></i>
+            </a>
+          </li>
+          <li class="nav-link">
+            <a
+              class="nav-link"
+              href="https://twitter.com/ElarabFouad"
+              rel="noreferrer noopener"
+            >
+              <i class="fab fa-twitter"></i>
+            </a>
+          </li>
+          <li class="nav-link">
+            <a class="nav-link" href="#" rel="noreferrer noopener">English</a>
+          </li>
+          <li class="nav-link">
+            <a class="nav-link" href="#" rel="noreferrer noopener">My Page</a>
+          </li>
+          <li class="nav-link">
+            <a class="nav-link" href="#" rel="noreferrer noopener">Logout</a>
+          </li>
+        </ul>
```

Comment:
- [ ] Currently, these social media links, logout, and language links are not visible as from width: 768px to 991px and the logo becomes very small. Let's fix this by making sure that these elements are visible once you get to the desktop breakpoint specified in your media query which is `min-width: 768px` . This should be effected on the `about` page too.

<img width="748" alt="Loom_7Cofq3SIUN" src="https://user-images.githubusercontent.com/81468675/152482592-ffcc5f89-5436-4efe-baa6-e7739fe29745.png">